### PR TITLE
DEV-3514-feat(piscine-go): add README.md for new go exercise reversemenuindex

### PR DESCRIPTION
**Pull request by tiagocollot**
Created and added README.md for new go exercise reversemenuindex

### CODE REVIEW

File: subjects/reversemenuindex/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+## reversemenuindex
+
+### Instructions
+
+The restaurant employees are having a really tough day and are delivering the customers' food in the wrong order. You need to fix the problem so that they can deliver it correctly.
+
+Write a function `ReverseMenuIndex` that takes an array of strings as arguments and returns another array of strings with the menu in the correct order.
+
+- `append` is not allowed for this exercise.
```
Comment (jotapero) [inline subjects/reversemenuindex/README.md:9]: write as `append()` instead, so student knows it's a function that exists

**Review by jotapero — CHANGES_REQUESTED** [review]

**Review by jotapero — APPROVED** [review]

**Review by jotapero — APPROVED** [review]

File: subjects/reversemenuindex/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+## reversemenuindex
+
+### Instructions
+
+The restaurant employees are having a really tough day and are delivering the customers' food in the wrong order. You need to fix the problem so that they can deliver it correctly.
+
+Write a function `ReverseMenuIndex()` that takes an array of strings as arguments and returns another array of strings with the menu in the correct order.
```
Comment (eslopfer) [inline subjects/reversemenuindex/README.md:7]: Wouldn't it be a slice of strings?

File: subjects/reversemenuindex/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+)
+
+func main() {
+  fmt.Println(piscine.ReverseMenuIndex([]string{"desserts", "mains", "drinks", "starters"}))
+}
+```
+
+And its output:
+
+```go
```
Comment (eslopfer) [inline subjects/reversemenuindex/README.md:38]: Wouldn't this be console instead of go?

**Review by eslopfer — CHANGES_REQUESTED** [review]

File: subjects/reversemenuindex/README.md (hunk 2 above)
Comment (tiagocollot) [reply subjects/reversemenuindex/README.md:7]: You are correct, thank you for this observation. It is fixed!

File: subjects/reversemenuindex/README.md (hunk 3 above)
Comment (tiagocollot) [reply subjects/reversemenuindex/README.md:38]: Yes, it should be console instead. It is fixed.

**Review by eslopfer — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**